src/modules/Builder/features/SaveAndPublish/SaveAndPublish.hooks.ts

```diff
@@ -307,8 +313,12 @@ export const useSaveAndPublishSetup = (): SaveAndPublishSetup => {
     if (responseStatus === 'loading' && checkIfAppletBeingCreatedOrUpdatedRef.current) {
       setPublishProcessStep(SaveAndPublishSteps.BeingCreated);
     }
-    responseStatus === 'error' && setPublishProcessStep(SaveAndPublishSteps.Failed);
-  }, [responseStatus, responseTypePrefix]);
+    if (isResponseStatusError) {
+      if (hasAccessDeniedError) return;
```

Comment:
Didn't get where is processed the `Access Denied` error ?

Reply:
This error is processed in the interceptor:
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/f36ecc76-5469-4902-aa53-72160de91740)

The return here is to avoid appearing two different popups.